src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -64,6 +67,10 @@ private void configureSubsystemDefaults() {}
   /** Configures trigger -> command bindings */
   private void configureBindings() {
     autonomous().whileTrue(new ProxyCommand(autos::get));
+    // operator.x().onTrue(shooter.shootCommand(1)); //key c
+    operator.x().onTrue(Commands.runOnce(() -> shooter.target += 1)); // key c              increase target by 1
```

Comment:
the way you're doing this is a little weird. Robot shouldn't be able to directly change the shooter target. Instead, you could have a method in Shooter that returns a command that increases/decreases the target speed by one. Better yet, you can have a method in Shooter that returns a command to set the target, and then have a default shooting speed in ShooterConstants. Or you could have a method in shooter that takes a distance away from the target and returns a command that sets the speed to some value that depends on the distance, if that makes sense.

Reply:
So I just read Shooter and it seems like you basically do have a method like the one I described. You can just bind something to the output of your shoot method